client/python/cli/options/parser.py

```diff
@@ -55,26 +55,25 @@ class Parser(object):
             str,
             hint="access token for token-based authentication",
         ),
+        Argument(
+            Arguments.REALM,
+            str,
+            hint="realm to use with header. if not defined will be used default realm from Polaris server. read more: https://polaris.apache.org/releases/1.1.0/configuration/",
+            default=None
+        ),
+        Argument(
+            Arguments.HEADER,
```

Comment:
When I suggested a `--header` flag, what I meant is that you could pass in any arbitrary header here such as `--header "X-foo: bar"`, not that the flag `--header` would be a good name for something which is just the realm context header name.